src/modules/selects/MultiSelect/MultiSelect.scss

```diff
@@ -15,10 +13,19 @@
             color: $hf-color-white;
 
             &-icon {
-                float: right
+                float: right;
             }
 
-            &-label {
+            &-icon, &-label {
+                font-size: $hf-font-size-base;
+                line-height: $hf-line-height;
+                padding: 2px 5px;
+            }
+        }
+        .Select-input {
+            height: 33px;
+
+            > input {
```

Comment:
probably like this maybe: `& > input {` ?

Reply:
And 2 more points:

1. It's always better to stack to class names that to tag itself;
2. In this particular case, it seems that CSS rules for `input` are not necessary